src/main/resources/dse_protocol_v1.spec

```diff
@@ -330,7 +330,8 @@ Table of Contents
 
   Asks the server to return which STARTUP options are supported. The body of an
   OPTIONS message should be empty and the server will respond with a SUPPORTED
-  message.
+  message. Since OPTIONS can come before a STARTUP request, as a consequence,
```

Comment:
I wouldn't change the specs here, these files are merely copies of the specs found in the Cassandra/DSE codebase.